include/z64actor.h

```diff
@@ -644,4 +644,10 @@ typedef struct {
     /* 0x24 */ char unk_24[0x4];
 } NpcInteractInfo; // size = 0x28
 
+#if OOT_DEBUG
+    #define ACTOR_PRINTF if (HREG(20) != 0) PRINTF
+#else
+    #define ACTOR_PRINTF(args) (void)0
+#endif
+
```

Comment:
id say this can go directly in the c file. the header is getting included in every actor and they dont really need this.
this is similar to `PLAY_LOG` in play.c

Reply:
The formatting is a bit weird but that's fine, it's been moved

Reply:
I also kinda think "ACTOR_PRINTF" is a bit broad for a name, makes it sounds a bit more general purpose than it is. It should probably tied to the name of the reg

`HREG(20)` -> `R_ENABLE_ACTOR_DEBUG_PRINTF`
and then `ACTOR_DEBUG_PRINTF()`?